datasets/inspecties/dataset.json

```diff
@@ -0,0 +1,280 @@
+            "type": "string",
+            "description": "Volledige naam van de inspecteur."
+          },
+          "photosPhotoUri": {
+            "type": "string",
+            "description": "Link naar de foto"
+          },
+          "xMeting": {
+            "type": "string",
+            "description": "X-coordinaat van meting (RD EPSG:28992)"
+          },
+          "yMeting": {
+            "type": "string",
+            "description": "Y-coordinaat van meting (RD EPSG:28992)"
+          },
+          "geometryCoordinates": {
+            "type": "string",
+            "description": "Coordinaten van de feature (RD EPSG:28992)"
+          },
+          "inspectionTypeId": {
+            "type": "string",
+            "description": "Id van de beeldmeetlat"
+          },
+          "inspectionTypeUri": {
+            "type": "string",
+            "description": "Link naar de beeldmeetlat info"
+          },
+          "inspectionTypeName": {
+            "type": "string",
+            "description": "Link naar de beeldmeetlat info"
+          },
+          "crowCode": {
+            "type": "string",
+            "description": "CROW unieke code van de beeldmeetlat (RAW code)"
+          },
+          "inspectionTypeOptionId": {
+            "type": "string",
+            "description": "Id van de beeldmeetlat optie"
+          },
+          "inspectionTypeOptionUri": {
+            "type": "string",
+            "description": "Link naar de beeldmeetlat optie"
+          },
+          "inspectionTypeOptionName": {
+            "type": "string",
+            "description": "Naam van de beeldmeetlat optie (bijv. A+)"
+          },
+          "TypeOptionDescription": {
+            "provenance": "inspection_type_option_description",
+            "type": "string",
+            "description": "Omschrijving van de beeldmeetlat optie"
+          },
+          "TypeOptionMeasuring": {
+            "provenance": "inspection_type_option_measuring",
+            "type": "string",
+            "description": "Meetinstructie van de beeldmeetlat optie, bevat html"
+          },
+          "cbsgridId": {
+            "type": "string",
+            "description": "Uniek identificatie van het object"
+          },
+          "cbsgridGeometrie": {
+            "type": "string",
+            "description": "Geometrie CBS rastercode (CRS28992RES100M)"
+          },
+          "buurtcodeDominant": {
+            "type": "string",
+            "description": "Unie­ke code (hier­in zie je de Stads­deel- en Wijk­co­de te­rug) (dominant) "
+          },
+          "buurtnaamDominant": {
+            "type": "string",
+            "description": "De naam van het ob­ject (dominant: De buurt waar het grootste gedeelte van de oppervlakte van het meetgebied ligt.)"
+          },
+          "buurtnaam": {
+            "type": "string",
+            "description": "De naam van het ob­ject"
+          },
+          "buurtcode": {
+            "type": "string",
```

Comment:
Als deze buurtcode dezelfde is als in de `gebieden.buurten` dataset, zou er een relatie van gemaakt kunnen worden: https://schemas.data.amsterdam.nl/datasets/gebieden/buurten/v1.1.1